assets/src/components/form/layouts/two-column-admin/two-column-admin.css

```diff
@@ -483,6 +483,10 @@
     margin-top: 3px;
 }
 
+.ee-two-column-admin .ee-visible-label {
+    padding-left: .4rem;
```

Comment:
@tn3rb I am not :100: % happy with this approach.
If there is a better mechanism of handling spacing in EE, please do suggest.
On the previous project, we had something similar to https://getbootstrap.com/docs/4.3/utilities/spacing/